docs/setup/independent/install-kubeadm.md

```diff
@@ -10,11 +10,12 @@ This page shows how to use install kubeadm.
 
 {% capture prerequisites %}
 
-* One or more machines running Ubuntu 16.04+, CentOS 7 or HypriotOS v1.0.1+
+* One or more machines running Ubuntu 16.04+, Debian 9, CentOS 7, RHEL 7, Fedora 25/26 (YMMV) or HypriotOS v1.0.1+
```

Comment:
nit: Based on the tone of this doc, I would expand YMMV.

Reply:
done